lib/rails_stats/console_formatter.rb

```diff
@@ -3,8 +3,6 @@
 module RailsStats
   class ConsoleFormatter < StatsFormatter
     def to_s
-      Bundler::Stats::CLI.start
```

Comment:
@etagwerker I saw that you added this in https://github.com/fastruby/rails_stats/pull/21

It was causing me trouble because Bundler was trying to interpret the first command line argument as a Bundler command.

The gem reporting seems to still work without it, so perhaps it was only needed for an older version of Bundler?

Reply:
Actually, I think it's not correctly counting the gems, I'll need to dig into this a bit more.

Reply:
This issue is the same as reported in https://github.com/fastruby/rails_stats/issues/30. I'm getting that too.